configuration.md

```diff
@@ -13,10 +13,14 @@ Casual supports the following configuration options which can be set using envir
     See [Inbound Startup Configuration](inbound.md#startup-configuration) for more details.
 * `CASUAL_OUTBOUND_USE_EPOLL` - If set to true, Netty uses epoll instead of NIO.
 * `CASUAL_INBOUND_USE_EPOLL` - If set to true, Netty uses epoll instead of NIO.
+* `CASUAL_INBOUND_STARTUP_INITIAL_DELAY_IN_SECONDS` - Delay the startup of the inbound server. Default is no delay.
 
 NB - if a `CASUAL_CONFIG_FILE` is provided, it take precedence over the `CASUAL_INBOUND_STARTUP_MODE` setting.
 However, if some configuration is missing from the configuration file but has a setting via an environment variable then this will be used before any hardcoded default setting.
 
+If `CASUAL_INBOUND_STARTUP_INITIAL_DELAY_IN_SECONDS` is configured then it will always delay the startup by that amount of seconds regardless of startup mode.
```

Comment:
Need to update the Inbound.md too, so as to make this clear in the different modes what happens.
i.e. IMMEDIATE actually with this setting will just immediately start waiting for the delay to complete.
TRIGGER will wait until the trigger app is registered and then delay.
DISCOVERY will wait until all the registered inbound services are found and then delay.
It's the first one which is the bad apple as it is contradictory - IMMEDIATE - but not quite as you want to delay! :|